packages/dialect-react-ui/components/CardinalAvatar/index.tsx

```diff
@@ -0,0 +1,53 @@
+import type { PublicKey, Connection } from "@solana/web3.js";
+import { useAddressImage } from '@cardinal/namespaces-components';
+import { HiUserCircle } from "react-icons/hi";
+
+export const CardinalAvatar = ({
+  connection,
+  address,
+  style,
+  height = "150px",
+  width = "150px",
+  dark = false,
+  placeholder,
+}: {
+  connection: Connection;
+  address: PublicKey | undefined;
+  height?: string;
+  width?: string;
+  dark?: boolean;
+  placeholder?: React.ReactNode;
+  style?: React.CSSProperties;
+}) => {
+  const { addressImage, loadingImage } = useAddressImage(connection, address);
+
+  if (!address) return <></>;
+  return loadingImage ? (
+    <div
+      style={{
+        ...style,
+        height,
+        width,
+        borderRadius: "50%",
```

Comment:
To be consistent on styles approach — it's better to use tailwind styles here, like `className="dt-rounded-full"`